src/indexExample.ts

```diff
@@ -35,8 +47,12 @@ class FancyLogger implements Logger {
     }
 }
 
-// Use a custom logger instead of the default console logger (this is optional!)
-setLogger(new FancyLogger());
+const logger = new FancyLogger();
+mariner.setLogger(logger);
+
+logger.info(`Input:  ${inputFilePath}`);
+logger.info(`Output: ${outputFilePath}`);
 
-const ddr = new DependencyDetailsRetriever();
+// Here is the code to actually call mariner
+const ddr = new mariner.DependencyDetailsRetriever();
 ddr.run(token, inputFilePath, outputFilePath);
```

Comment:
Perhaps we should add `.then(() => logger.info('Done!')` here to make it clear run returns a promise.